packages/python/plotly/_plotly_utils/tests/validators/test_fig_deepcopy.py

```diff
@@ -0,0 +1,11 @@
+import copy
+import plotly.express as px
+
+
+def test_deepcopy():
+    gapminder = px.data.gapminder()
+    fig = px.line(gapminder, x="year", y="gdpPercap", color="country")
+
+    fig_copy = copy.deepcopy(fig)
+
+    assert fig_copy is not None
```

Comment:
This test looks good thank you for doing that! Do you think you could add a simple check that the data did copy correctly? Also, where are the other deepcopy tests you mentioned? Does it make more sense to include this test with those?

Reply:
Thanks @marthacryan , what's the best way to assert that the values are the same? is it `fig_copy.to_dict() == fig.to_dict()`?

Here are few other figure deepcopy's in the test suite:
- [test_update_subplots](https://github.com/plotly/plotly.py/blob/725290068d5ea3f7f344eec7d7b84c58b85cfb38/packages/python/plotly/plotly/tests/test_core/test_update_objects/test_update_subplots.py#L364)
- [test_update_traces](https://github.com/plotly/plotly.py/blob/725290068d5ea3f7f344eec7d7b84c58b85cfb38/packages/python/plotly/plotly/tests/test_core/test_update_objects/test_update_traces.py#L295)
- [test_deepcopy_pickle.py](https://github.com/plotly/plotly.py/blob/725290068d5ea3f7f344eec7d7b84c58b85cfb38/packages/python/plotly/plotly/tests/test_io/test_deepcopy_pickle.py#L38)

Reply:
I think using `fig.to_dict()` is fine here. There are other ways to compare them but I just wanted a baseline check that the data is still there